tests/helpme/test_create.py

```diff
@@ -343,3 +344,220 @@ def test_determinant_spline(
             det_spl.evaluate(queries=rnd_queries).ravel(),
             np.linalg.det(sp_i.jacobian(queries=rnd_queries)),
         ), f"{sp_i.whatami} at index {idx} failed determinant spline"
+
+
+### TESTS FOR SWEEPING ###
+
+
+# test the basic functionality of the swept-function
+def test_swept_basic_functionality():
+    cross_section = splinepy.NURBS(
```

Comment:
isn't this one same as a fixture we already have?

Reply:
I can't find a similar fixture.